code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -126,7 +140,11 @@
 					S.faction |= "slime"
 				playsound(user, 'sound/effects/splat.ogg', 50, 1)
 				user.visible_message("<span class='warning'>[user] spits out [S]!</span>", "<span class='warning'>You spit out [S]!</span>")
+				activating = FALSE
 				return 600
+			else
+				activating = FALSE
+				return 0
```

Comment:
`return FALSE`

Reply:
These are cooldowns, I don't feel like false is appropriate here.

Reply:
you're using 0. you should use FALSE